src/reader/config.rs

```diff
@@ -136,13 +136,13 @@ impl ParserConfig {
     /// let mut source: &[u8] = b"...";
     ///
     /// let reader = ParserConfig::new()
-    ///     .add_entity("nbsp", ' ')
-    ///     .add_entity("copy", '©')
-    ///     .add_entity("reg", '®')
+    ///     .add_entity("nbsp", " ")
+    ///     .add_entity("copy", "©")
+    ///     .add_entity("reg", "®")
     ///     .create_reader(&mut source);
     /// ```
-    pub fn add_entity<S: Into<String>>(mut self, entity: S, value: char) -> ParserConfig {
-        self.extra_entities.insert(entity.into(), value);
+    pub fn add_entity<S: Into<String>>(mut self, entity: S, value: &str) -> ParserConfig {
```

Comment:
Please make the `value` also generic with the `: Into<String>` bound. It would probably be also good to extract the bounds to a `where` block.